201-vmss-azureautomationdsc/README.md

```diff
@@ -0,0 +1,62 @@
+
+# VM Scale Set Configuration managed by Azure Automation DSC
+
+[![Deploy to Azure](http://azuredeploy.net/deploybutton.png)](https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2Fmgreenegit%2FARM-ScaleSetmanagedbyAADSC%2Fmaster%2Fazuredeploy.json)
+
+This repo serves to prove an ARM template to deploy a VM Scale Set where virtual machines are deployed as registered nodes in the Azure Automation Desired State Configuration service, and node configuration is guaranteed consistency after deployment, and the AADSC service components are provided in the same deployment template.
+
+The Azure Resource Manager template includes:
+- Deploy virtual machines in Scale Set with autoscale rules defined
+- Distribute VHD files across 5 storage accounts
+- Configure Azure Automation DSC service with configuration and modules to manage the virtual machines
```

Comment:
Maybe tell them what the machines will be configured to look like so they can see it worked once they deploy this?